RSDKv5/RSDK/User/Core/UserCore.cpp

```diff
@@ -500,7 +500,7 @@ void RSDK::LoadSettingsINI()
 
 #if !RETRO_USE_ORIGINAL_CODE
         customSettings.region                    = -1;
-        customSettings.confirmButtonFlip         = false;
+        customSettings.confirmButtonFlip         = true;
```

Comment:
Maybe keep the original value using `#if`.
```suggestion
#if RETRO_PLATFORM = RETRO_WII
        customSettings.confirmButtonFlip         = true;
#else
        customSettings.confirmButtonFlip         = false;
#endif
```